.travis.yml

```diff
@@ -64,7 +64,7 @@ script:
   - yarn install --frozen-lockfile
   - bundle exec rails db:create
   - bundle exec rails db:schema:load
-  - bundle exec rails webpacker:compile
+  - bundle exec rails assets:precompile
```

Comment:
This appears to fix the flakiness of the test server that runs before end to end tests run. I think this change makes sense as it also mimics production more closely. It does add some more time to a build though.

This adds to the work I did in #11977. Thoughts on this change @rhymes @citizen428?

Reply:
It looks like it's adding around 15 seconds to the build. I checked some other PRs and `webpacker:compile` takes around 15 seconds, this takes double that. Hopefully we can regain that time when we convert system tests to Cypress